core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.ts

```diff
@@ -0,0 +1,65 @@
+import { of } from 'rxjs';
+
+import { Component, ElementRef, EventEmitter, Output, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, FormControl, Validators } from '@angular/forms';
+
+import { catchError } from 'rxjs/operators';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+interface FormValues {
+    textPrompt: string;
+}
+
+@Component({
+    selector: 'dot-ai-content-prompt',
+    templateUrl: './ai-content-prompt.component.html',
+    styleUrls: ['./ai-content-prompt.component.scss']
+})
+export class AIContentPromptComponent {
+    isFormSubmitting = false;
+
+    @ViewChild('input') input: ElementRef;
+
+    @Output() formValues = new EventEmitter<FormValues>();
+    @Output() formSubmission = new EventEmitter<boolean>();
+
+    form = new FormGroup({
+        textPrompt: new FormControl('')
+    });
```

Comment:
```suggestion
    form: FormGroup<AIContentForm> = new FormGroup<AIContentForm>({
        textPrompt: new FormControl('', Validators.required)
    });
```

The interface should be:

```ts
interface AIContentForm {
    textPrompt: FormControl<string>;
}
```

Also, you can remove the `buildForm` method